infra/base-images/base-runner/run_fuzzer

```diff
@@ -151,6 +153,27 @@ elif [[ "$FUZZING_ENGINE" = honggfuzz ]]; then
   # -n: number of fuzzing threads (and processes)
   CMD_LINE="$OUT/honggfuzz -n 1 --exit_upon_crash -V -R /tmp/${FUZZER}_honggfuzz.report -W $FUZZER_OUT -v -z -P -f \"$CORPUS_DIR\" $(get_dictionary) $* -- \"$OUT/$FUZZER\""
 
+elif [[ "$FUZZING_ENGINE" = centipede ]]; then
+
+  # Create the work and corpus directory for Centipede.
```

Comment:
This part is not ready yet: I will push a few commits to 1) edit the comment, and 2) organise the dirs.

Reply:
Is this part still not ready?

Reply:
It is now : )
Will come back and enable `address_space_limit_mb` after [this issue](https://github.com/google/centipede/issues/166) is fixed.